Formula/cargo-bloat.rb

```diff
@@ -0,0 +1,24 @@
+class CargoBloat < Formula
+  desc "Find out what takes most of the space in your executable"
+  homepage "https://github.com/RazrFalcon/cargo-bloat"
+  url "https://github.com/RazrFalcon/cargo-bloat/archive/v0.10.1.tar.gz"
+  sha256 "e8acd89ffcd555934672b0a10ba3b64072eaabf9d4ebe6be1bb6d4d7c0f9f73c"
+  license "MIT"
+  head "https://github.com/RazrFalcon/cargo-bloat.git", branch: "master"
```

Comment:
```suggestion
  head "https://github.com/RazrFalcon/cargo-bloat.git"
```

Reply:
I added it because of `brew audit` error:

```bash
❯ brew audit --new cargo-bloat
cargo-bloat:
  * HEAD: Use `branch: "master"` to specify the default branch
Error: 1 problem in 1 formula detected
```

Reply:
yeah, you are right, looks like we updated the audit rule now.